docs/test.md

```diff
@@ -1,7 +1,17 @@
 # Biblioteca de aserciones y de framework de test
 
 ## Biblioteca de aserciones
-Para las assert se ha decidido usar la bilioteca que viene por defecto con python además de assertpy la cual me permite unas opciones más que necesito para validar mi código, con afirmaciones simples para pruebas unitarias en python y también con la que puedo escalar a futuro los assert del mismo, esta elección se ha realizado en base a búsquedas en diferentes web, la popularidad y uso de la comunidad de esta, su populariad radica en que se descangan una media de 35.340 descargas a la semana, la comunidad se puede observar en esta [web](https://snyk.io/advisor/python/assertpy) y fijandonos en su [repositorio de github](https://github.com/assertpy/assertpy) la cual se observa una comunidad activa, cuenta con numerosas estrellas en github (329) y númerosos contribuidores, en base a esto se decidió ver esta biblioteca de aserciones en primera instancia, luego leyendo la extensa [documentación](https://assertpy.github.io/docs.html) se pudo comprobar que se ajustaba a las necesidades básicas primarias que necesitaba mi código.
+
+Para ello se proponen una serie de criterios que ha de cumplir nuestra biblioteca para nuestro proyecto:
+
+* Un buen soporte, ya que queremos, poder resolver cualquier duda de manera rápida, es decir que tenga una comunidad activa, debido a la caracteristica de nuestro proyecto de c caracter docente, en el que se aprende, en un tiempo determinado, un cuatrimestre.
```

Comment:
Esto lo ponéis todos. Si, de forma realista, vuestro proyecto va a durar lo que la asignatura, no os hace falta el soporte, y esto se acaba convirtiendo en una excusa para elegir las bibliotecas "populares", que son las que elige todo el mundo.